apps/judicial-system/api/src/app/app.module.ts

```diff
@@ -68,6 +69,7 @@ const autoSchemaFile = production
     CaseListModule,
     DefendantModule,
     DefenderModule,
+    SubpoenaModule,
```

Comment:
Retrieving subpoena updates has been moved from the police module to a new subpoena module. When needed the client gets the lates subpoena from the server and the backend takes care of querying the police system for subpoena updates if required..